sklearn/model_selection/_validation.py

```diff
@@ -1089,6 +1089,12 @@ def learning_curve(estimator, X, y, groups=None,
         Whether to shuffle training data before taking prefixes of it
         based on``train_sizes``.
 
+    stratify : boolean, optional
+        Whether to stratify training data before taking prefixes of it,
+        :class:`StratifiedShuffleSplit` is used.
```

Comment:
I think mentioning StratifiedShuffleSplit only confuses the relationship with`cv`.

I would rather it explained as "when True, each training set will be sampled such that the proportion of each class matches the full training data as well as possible"

I'm not sure this needs shuffling to work, either. You might be able to stable-sort the indices by y, then remove indices at a roughly-fixed interval. This may not always ensure the inclusion of very rare classes (but I'm not sure StratifiedShuffleSplit does either).